docs/_documentations/troubleshooting.md

```diff
@@ -398,21 +398,21 @@ Issue link: https://github.com/eclipse/codewind/issues/1061
 -->
 ## Appsody binds fail
 
-If an Appsody respository is already added in your local Appsody CLI, the Appsody bind might fail in the following scenario:
-1. Use the template manager to add the Appsody repository in Codewind.
+If an Appsody repository is already added in your local Appsody CLI, the Appsody bind might fail in the following scenario:
```

Comment:
Looks good.  The only other comment here is the points below seem to be written as a sequence of steps, but they can in fact happen far apart in time.  They are more like conditions that have happened in order for this problem to occur.

Reply:
@makandre If the events are sequential, regardless of the amount of time between them, I think it's good to keep the numbers. :) However, if the events can happen in any order, then we should omit the numbers.

Reply:
I'm not against the number.  But could we say something like:  You have at one time added a repo from your IDE, then later you try to bind... etc?

Reply:
@makandre I tried to edit the text so that it indicates that the events are sequential but could have happened with time between them. Please see the latest commit for the changes. Thanks!